File: Lab Assignment 6/Task2.py
def insertandsort(list,i):
    if i<1:
        return None
    insertandsort(list,i-1)
    temp = list[i-1]
    j = i-2
    while j>=0 and list[j]>temp:
        list[j+1] = list[j]
        j = j-1
    list[j+1] = temp
list = [3,1,2,5,6,7,4]
insertandsort(list,len(list))
print(list)

File: Lab Assignment 1/Task2.java
/// Task__2///
import java.util.Scanner;
public class Task2 {
    public static void main(String []args){
        int [] source = {10,20,30,40,50,60};
        Scanner sc= new Scanner (System.in);
        System.out.print("Enter the value of k: ");
        int k=sc.nextInt();
        rotateLeft(source,k);
        sc.close();
    }
    public static void rotateLeft(int []source, int k){
        for (int i=0, j=i+k;i<k;i++,j++){
            int temp= source[i];
            source[i]=source[j];
            source[j]=temp;
        }
        for(int i=0;i<source.length;i++){
            System.out.print(source[i]+",");
        }
    }
}
